widgetssdk/src/main/java/com/glia/widgets/core/notification/NotificationRemovalService.kt

```diff
@@ -5,6 +5,14 @@ import android.app.Service
 import android.content.Intent
 import android.os.IBinder
 
+/**
+ * Glia internal class.
+ *
+ * Will be automatically added to integrator Manifest by Manifest merger during compilation.
+ *
+ * This service is used to remove push notification about ongoing video/audio call when application is killed.
+ * Related bug ticket: https://glia.atlassian.net/browse/MOB-2481
```

Comment:
It seems to me we don't need this bug ticket reference here because it's our internal information.